spec/runner.js

```diff
@@ -1,24 +1,28 @@
-const { execSync } = require("child_process");
+const { execSync } = require('child_process');
 
-function run(command){
+function run(command) {
     let output = null;
     let err = null;
     let statusCode = 0;
 
-    try{
+    try {
         output = execSync(command);
     }
-    catch(error){
+    catch(error) {
         statusCode = error.status;
         err = error;
     }
-    finally{
-        return { error: decodeUTF8(err), data : decodeUTF8(output), status : statusCode};
+    finally {
+        return { 
+            error: decodeUTF8(err), 
+            data : decodeUTF8(output), 
+            status : statusCode
+        };
     }
 }
 
-function decodeUTF8(decode){
-    return (!!decode) ? decode.toString('utf8') : null;
+function decodeUTF8(decode) {
+    return decode.toString('utf8') || null;
```

Comment:
If `decode` is `null` the following error will be thrown.
`TypeError: Cannot read property 'toString' of null`. Because of that, we have to check if `decode` is null.

Reply:
Ah.. I was just editing things with GitHub editor. let's keep the ternary operator as you used before ;) Thanks